### ISSUE TITLE
Low focal and distortion accuracy when training

### ISSUE THREAD

**Issue by LeonhardEulerr** [alexvbogdan/DeepCalib#27]
Hi,

What was your `output_distortion_accuracy` and `output_focal_accuracy` after model has been trained on entire dataset? 

When I train the Classification SingleNet  model on similar dataset I get very low accuracy (~0.02). 

When training/fine-tuninig the model after loading your weights, the accuracy is about the same (~0.02) after each epoch.

Would appreciate if you gave any hint on what can cause such low acc.

**Comment by alexvbogdan** [comment]
Hi, it is hard to tell you what exactly caused such a low accuracy without knowing some more details. What kind of training data have you used and how many training samples have you used? What hyperparameters are you using during the training (learning rate, number of epochs, etc.)? Please provide as many details as possible, anything you would consider useful and I will try to help you with your problem.

**Comment by LeonhardEulerr** [comment]
Hi @alexvbogdan,
I have used synthetically created panoramas that were not so good and I gues the model had hard time extracting some features.

Then I have used the dataset that you published (~500 FHD panoramas) and augmented it so that I get around 50 images out of one panorama. I have used all the hyperparameters as in your code. The classification is far from good but regression works well on this amount of data. 

I am aware of fact that you have used much greater number of high quality panoramas so I guess the problem might be in amount of data.

Thanks for the reply :)

**Comment by alexvbogdan** [comment]
Yes, we used way more data, there were around 67000 panoramas compared to the 500 that you use. After generating training dataset we had over a million samples. You can try another backbone architecture, we used InceptionV3 which is quite large, I would suggest mobilenets or some efficientnets with a low number of parameters for your amount of data.
Please let me know if that helps.

**Comment by LeonhardEulerr** [comment]
I actually experimented with smaller nets and used mobilenet v2 but as I decided to go for regression only that worked for me on this amount of data, I haven’t tested classification anymore. 

If I will ever give it a go I will update you :)
